### PR TITLE
feat(Commands/meme.ts): Improve meme command NSFW filter

### PR DESCRIPTION
**Pull request by khiromoshito**

### CODE REVIEW

File: src/commands/fun/meme.ts [hunk 1]
```diff
@@ -17,16 +17,21 @@ class Meme extends Command {
 	}
 
 	async run(message: Message) {
-		const response = await (await fetch('https://meme-api.herokuapp.com/gimme/memes/2')).json();
-		const memesArray: MemeResponse[] = response.memes.filter((meme: MemeResponse) => meme.nsfw === false);
-		const memeData: MemeResponse = memesArray[0];
+
+		// Returns 10 memes from the API, which would give a 99.90% chance of getting a non-NSFW meme
```
Comment (khiromoshito) [inline src/commands/fun/meme.ts:21]: **FUN FACT**: This is mathematically accurate.

Comment (Megatank58) [reply src/commands/fun/meme.ts:21]: It will increase the size of the memes fetched though?

Comment (khiromoshito) [reply src/commands/fun/meme.ts:21]: Sure you can decrease the meme count. 5 can do, with 96.88% chance.

Comment (Megatank58) [reply src/commands/fun/meme.ts:21]: If you dont mind how are the percentages calculated

**Review by Megatank58 — APPROVED** [review]

File: src/commands/fun/meme.ts (hunk 1 above)
Comment (khiromoshito) [reply src/commands/fun/meme.ts:21]: `(1−(1÷(2^numberOfMemes)))×100`

**Pull request merged**